packages/manager/apps/pci-volume-backup/src/pages/create/Create.page.tsx

```diff
@@ -0,0 +1,377 @@
+import {
+  PciDiscoveryBanner,
+  useProject,
+  useVolumes,
+} from '@ovh-ux/manager-pci-common';
+import {
+  BaseLayout,
+  Notifications,
+  PciGuidesHeader,
+  useNotifications as useMRCNotifications,
+  useProjectUrl,
+} from '@ovh-ux/manager-react-components';
+import {
+  PageType,
+  ShellContext,
+  useOvhTracking,
+} from '@ovh-ux/manager-react-shell-client';
+import {
+  OdsInputChangeEventDetail,
+  OdsInputCustomEvent,
+  OdsSelectChangeEventDetail,
+  OdsSelectCustomEvent,
+} from '@ovhcloud/ods-components';
+import {
+  OdsBreadcrumb,
+  OdsBreadcrumbItem,
+  OdsButton,
+  OdsLink,
+  OdsMessage,
+  OdsText,
+} from '@ovhcloud/ods-components/react';
+import { formatISO } from 'date-fns';
+import { Suspense, useCallback, useContext, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Outlet, useHref, useNavigate, useParams } from 'react-router-dom';
+import { VOLUME_BACKUP_TRACKING } from '@/tracking.constant';
+import { useNotifications } from '@/hooks/notifications/useNotifications';
+import {
+  useCreateVolumeBackup,
+  useCreateVolumeSnapshot,
+} from '@/data/hooks/useVolumeBackup';
+import {
+  BACKUP_NAME_PREFIX,
+  VOLUME_OPTION_BACKUP,
+  VOLUME_OPTION_SNAPSHOT,
+} from '@/constants';
+import BackupNameStep from './steps/BackupNameStep';
+import BackupOptionStep, { TBackupOption } from './steps/BackupOptionStep';
+import DetachVolumeStep from './steps/DetachVolumeStep';
+import VolumeSelectStep from './steps/VolumeSelectStep';
+
+const DEFAULT_BACKUP_OPTIONS: TBackupOption[] = [
+  {
+    id: 'volume_snapshot',
+    planCode: 'volume.snapshot.consumption',
+    type: VOLUME_OPTION_SNAPSHOT,
+    isDisabled: true,
+    price: null,
+  },
+  {
+    id: 'volume_backup',
+    planCode: 'volume-backup.storage.hour.consumption',
+    type: VOLUME_OPTION_BACKUP,
+    isDisabled: true,
+    price: null,
+  },
+];
+
+export default function CreateVolumeBackup() {
+  const { t } = useTranslation(['create', 'pci-volume-backup']);
+
+  const { projectId } = useParams();
+  const navigate = useNavigate();
+  const { trackClick, trackPage } = useOvhTracking();
+  const { shell } = useContext(ShellContext);
+
+  const hrefProject = useProjectUrl('public-cloud');
+  const blockStorageUrl = `${hrefProject}/storages/blocks`;
+  const backupsHref = useHref('..');
+
+  const { addSuccessMessage, addErrorMessage } = useNotifications({
+    ns: 'create',
+  });
+  const { clearNotifications } = useMRCNotifications();
+
+  const [selectedVolumeId, setSelectedVolumeId] = useState<string>('');
+  const [selectedBackup, setSelectedBackup] = useState<TBackupOption>();
+  const [backupName, setBackupName] = useState('');
+  const [backupOptions, setBackupOptions] = useState<TBackupOption[]>(
+    DEFAULT_BACKUP_OPTIONS,
+  );
+
+  const goBack = () => navigate('..');
+
+  const { data: project } = useProject();
+  const { data: volumes } = useVolumes(projectId || '');
```

Comment:
```suggestion
  const { data: volumes } = useVolumes(projectId);
```

If useVolumes cannot take undefined. Maybe the type is not defined. Have you set enabled to do not send request is value is not a set ?